### PR TITLE
Erigon 2021-08-01

### PR DESCRIPTION
**Pull request by d-xo**
###### Motivation for this change

- `turbo-geth` has been renamed to `erigon` ([ref](https://github.com/ledgerwatch/erigon/pull/2018))
- updates to the [latest release](https://github.com/ledgerwatch/erigon/releases/tag/v2021.06.03)

I'm not sure what the procedure for handling a name change in a package is? Should I leave a stub package that throws an error message at build time (e.g. "turbo-geth has been renamed to erigon") under `turbo-geth`, or is a changelog entry enough?

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 126750` at 632c3373 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>erigon</li>
    <li>nixos-install-tools</li>
  </ul>
</details>

File: nixos/doc/manual/release-notes/rl-2111.section.md [hunk 1]
```diff
@@ -20,6 +20,11 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 ## Backward Incompatibilities
 
+* The `erigon` ethereum node has moved to a new database format in `2021-05-04`, and
+  requires a full resync
+
+* The `turbo-geth` package has been renamed to `erigon`
+
```
Comment (SuperSandro2000) [inline nixos/doc/manual/release-notes/rl-2111.section.md:27]:
```suggestion

```

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -28125,8 +28127,6 @@ in
 
   tessera = callPackage ../applications/blockchains/tessera.nix { };
 
-  turbo-geth = callPackage ../applications/blockchains/turbo-geth.nix { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:28128]: Please add a throw entry to aliases.nix instead.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by d-xo** [comment]
Apologies for the delay, spaced out on this. Addressed review comments and updated to the latest version.

**Comment by SuperSandro2000** [comment]
Please resolve the merge conflict.

**Pull request merged**